libs/application/ui-shell/src/hooks/useDocumentTitle.ts

```diff
@@ -0,0 +1,19 @@
+import { useEffect, useRef } from 'react'
```

Comment:
I guess I could have use resolution on the package json to fix the react version. It was failing tests for I believe a mismatch of react versions